src/plugins/github/parseReferences.js

```diff
@@ -13,26 +13,21 @@ function findAllMatches(re: RegExp, s: string): any[] {
   return matches;
 }
 
-export function findNumericReferences(body: string): number[] {
-  return findAllMatches(/(?:\W|^)#(\d+)(?:\W|$)/g, body).map((x) => +x[1]);
+export function findAllReferences(body: string): string[] {
+  return findNumericReferences(body)
```

Comment:
`concat` is variadic. Prefer either of the following:
```js
return [].concat(
  findNumericReferences(body),
  findUsernameReferences(body),
  findGithubUrlReferences(body)
);
return [
  ...findNumericReferences(body),
  ...findUsernameReferences(body),
  ...findGithubUrlReferences(body),
];
```

Reply:
Done